value_converter.go

```diff
@@ -0,0 +1,44 @@
+package clickhouse
+
+import (
+	"database/sql/driver"
+	"reflect"
+	"strconv"
+)
+
+func (stmt *stmt) ColumnConverter(idx int) driver.ValueConverter {
+	return converter{}
+}
+
+func (c *conn) CheckNamedValue(nv *driver.NamedValue) (err error) {
+	nv.Value, err = converter{}.ConvertValue(nv.Value)
+	return
+}
+
+type converter struct{}
+
+func (c converter) ConvertValue(v interface{}) (driver.Value, error) {
```

Comment:
Please add tests to prove that your code works and the big uint64 is supported properly now.